### DISCUSSION TITLE
What library/SDK should I use for headless Node app?

### DISCUSSION THREAD

**Discussion by punto-** [WalletConnect/walletconnect-monorepo#2528] [Developer Support]
Hello, I need to make a headless Node app that receives request from another application (a game) to connect/interact with a wallet, the game shows the QR and all the UI necessary, the app only needs to manage the wallet connection, what library/sdk should I be using? I don't understand if web3modal is UI only or if it can be used from a headless app. Thanks

**Comment by junaama** [comment]
@punto- Did you ever figure this out?

**Comment by daplcor** [comment]
You can use the sign-api for this.  There is no need to introduce the web UI libraries.
